src/converter/internal/compat/backendapi.cpp

```diff
@@ -747,6 +749,8 @@ Ret BackendApi::applyTranspose(const INotationPtr notation, const std::string& o
         return make_ret(Ret::Code::InternalError);
     }
 
+    interaction->selectAll();
```

Comment:
This is basically what Mu3 does too, in file.cpp, `MuseScore::exportTransposedScoreToJSON`